### PR TITLE
Security Document

### PR DESCRIPTION
**Pull request by Yus147**

### CODE REVIEW

File: docs/source/Security/Security.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+We want to ensure that we do not expose our application or resources to ports, protocols or applications from where they do not expect any communication. Thus, minimizing the possible points of attack and letting us concentrate our mitigation efforts. 
+
+   Placing our computation resources behind `CDNs <https://en.wikipedia.org/wiki/Content_delivery_network>`_ or `Load Balancers <https://en.wikipedia.org/wiki/Load_balancing_(computing)>`_ and restricting direct Internet traffic to certain parts of our infrastructure. Or we can just use firewalls or ACLs to filter our site in/out traffic.
+
+*Since we do not know the detail on bandwidth capacity and server capacity. I'll look into it further once we have been informed. However, there is this* `open-source DoS protection <https://github.com/AltraMayor/gatekeeper/wiki>`_ *repo that I think could be useful*
+
+
+Secure RethinkDB Cluster
+------------------------
+https://rethinkdb.com/docs/security/
```
Comment (McSinyx) [inline docs/source/Security/Security.rst:20]: Please elaborate on this.

**Review by McSinyx — CHANGES_REQUESTED** [review]
CDN is cool, any idea how to use it?  As for load balancer, please see previous discussions (I'm too tired to look them up right now).  Firewall seems like a bright idea too.  ACL are already implemented in our persistency nodes IIUC though.  You'd need to look into Gatekeeper and provide a TL;DR on what it does and how to use it too.

Also what are other possible attacks?  In other words, you're doing this too late and too little to provide understanding to other team members.  Please try to submit as early as you can next time to make time for review and discussion.

**Comment by Huy-Ngo** [comment]
Please elaborate more. DDoS is not the only security vulnerability.

**Comment by DDHLong** [comment]
@Yus147 nitpick: branch name should be in lowercase

**Comment by McSinyx** [comment]
@Yus147, I highly recommend you put some effort reworking this.

**Comment by McSinyx** [comment]
Most DoS mitigation measures are carried out by either [Hypercorn](https://pgjones.gitlab.io/hypercorn/discussion/dos_mitigations.html) or [Quart](https://pgjones.gitlab.io/quart/discussion/dos_mitigations.html), except for data dribble (yes, a simple curl shows acanban.ga is over HTTP/2).  Other parts are addressed via configurations so I guess it is safe to close this and its linked issue for now.

**Pull request closed without merging**